client/verta/docs/tutorials/deployment/endpoint_update_config.rst

```diff
@@ -0,0 +1,79 @@
+Updating endpoint with a configuration file
+===========================================
+
+.. TODO: link to Endpoint.update tutorial
+
+Verta's client and CLI support updating Endpoint with JSON or YAML configuration file. This tutorial will show how it can be done, using the client and CLI.
+
+Configuration file
+------------------
+
+Sometimes, it is more convenient to specify update configuration in a separate, human-readable file, than to have everything in a Python script or a split CLI command.
+
+For example:
+
+.. code-block:: json
+
+        {
+            "run_id": "<experiment run id>",
+            "strategy": "canary",
+            "canary_strategy": {
+                "progress_step": 0.05,
+                "progress_interval_seconds": 30,
+                "rules": [
+                    {
+                        "rule": "latency_avg_max",
+                        "rule_parameters": [
+                            {
+                                "name": "threshold",
+                                "value": "0.1"
+                            }
+                        ]
+                    },
+                    {
+                        "rule": "error_4xx_rate",
+                        "rule_parameters": [
+                            {
+                                "name": "threshold",
+                                "value": "1"
+                            }
+                        ]
+                    }
+                ]
```

Comment:
I _think_ we should make this format more similar to the autoscaling metrics below. The fields are analogous, so we could make this more compact
```suggestion
                "rules": [
                    {"rule": "latency_avg_max", "rule_parameters": [{"name": "threshold", "value": "0.1"}]},
                    {"rule": "error_4xx_rate", "rule_parameters": [{"name": "threshold", "value": "1"}]}
                ]
```

Reply:
Done. Also users don't have to put the value in string.